### PR TITLE
Gildas: update to 201805a

### PR DESCRIPTION
**Pull request by bardeau**
@smaret can you please merge this PR for the monthly release? Thanks.

### CODE REVIEW

File: science/gildas/Portfile [hunk 1]
```diff
@@ -30,8 +30,8 @@ master_sites        http://www.iram.fr/~gildas/dist/ \
                     http://www.iram.fr/~gildas/dist/archive/gildas/
 distname            ${name}-src-${my_version}
 
-checksums           rmd160  ca1ec4aef259cda4e896110ece75d7ddb14d7db6 \
-                    sha256  e95edfcda46d8976d6dd2839578b95d0ec335fc69cf5e6b250ebb533b606f72c
+checksums           rmd160  fd20e215480cc4339f4200085dfdfbdbf85ca1bf \
+                    sha256  3c747d90fbd30e2e1b9d94babfd3da3b896025d814967390aee62b5bac8120f6
```
Comment (mf2k) [inline science/gildas/Portfile]: Please add the `size` tag to checksums.

Comment (bardeau) [reply science/gildas/Portfile]: Can you please give some reference? I do not see such tag on the official documentation for Portfiles: https://guide.macports.org/chunked/development.creating-portfile.html

Comment (mf2k) [reply science/gildas/Portfile]: It is the size of the download file in bytes. You can look at other Portfiles that use it like `py-numpy`. To calculate the value, just set it to 1 initially and use `port -v checksum` to get the correct value.

**Comment by pmetzger** [comment]
@bardeau I didn't realize you were one of the maintainers because your github ID isn't in the "maintainers" line for the port. Could you please add your ID to your email address (in the same way that @smaret is listed) to this pull request and squash your commits? I'll then merge the PR.

**Comment by bardeau** [comment]
@pmetzger I have done the commit you requested on the branch.

**Comment by pmetzger** [comment]
@bardeau Though you didn't squash it. That's okay in this case, but in the future, please squash commits when you can.

**Comment by pmetzger** [comment]
Merged, and thank you for your contributions to MacPorts!

**Comment by bardeau** [comment]
@pmetzger sorry I missed the squash part. I am not used yet to the github interface. As my commits in the branch are automatically added to the PR I thought this was enough.

**Pull request merged**